src/components/data/DataTable.js

```diff
@@ -103,7 +103,7 @@ const DataTable = props => {
   const signal = useCancellation()
 
   // Helpers
-  const loadData = _.flow(
+  const loadData = entityMetadata && _.flow(
```

Comment:
```suggestion
  const loadData = !!entityMetadata && _.flow(
```

to be more explicit about the intention of coercing `entityMetadata` into a boolean (assuming `entityMetadata` does not boolean values).